README.md

```diff
@@ -13,29 +13,32 @@ This repository contains the materials and instructions for the PySAL workshop
 at [NARSC 2019](http://www.narsc.org/newsite/wp-content/uploads/2019/06/pysal.pdf).
 
 
-## Schedule
+## Schedule 
 
+Wednesday, Nov 13, 2019
 
-* 9:00-10:40
+
+* 8:00-9:40
   * Overview of PySAL and workshop
   * Introductions
   * Setup
   * Jupyter notebooks
   * Python primer
-* 10:40-11:15
+* 9:40-10:15
   * Coffee Break
-* 11:15-13:00
+* 10:15-12:00
   * Spatial data processing
   * Choropleth mapping and geovisualization
-* 13:00-14:00
+* 12:00-13:00
   * Lunch
-* 14:00-15:45
+* 13:00-14:45
   * Spatial weights
   * Global spatial autocorrelation
   * Local spatial autocorrelation
-* 15:45-16:15
+* 14:45-15:15
   * Coffee Break  
-* 16:15-17:30
+* 15:15-17:00
   * Spatial inequality dynamics
   * Regionalization and clustering
+  * Basic spatial regression
```

Comment:
Do you want to add the content for this?

Reply:
I just deleted it.